reliquery/relic.py

```diff
@@ -58,6 +56,11 @@ def __init__(
             self._ensure_exists()
 
         self.metadata_db = MetadataDB()
+        self.relic = self.metadata_db.sync_relic_data(
```

Comment:
should this field be relic_data or even data? it seems confusing to have a field called relic on relic.